pointers/pointers.c

```diff
@@ -66,7 +79,11 @@ int string_compare(char *m, char *n)
 */
 char *find_string(char *haystack, char *needle)
 {
-
+    int y = 0;
+    for(int i = 0; &haystack[i] != &needle; i++){
+        y = i;
+    }
+    return haystack[y];
```

Comment:
Good start on the `needle in the haystack`. Make sure you stay aware of what data you're working with: memory locations, memory values, or copies in the right circumstances.